operator/builtin/input/udp/udp.go

```diff
@@ -62,11 +84,24 @@ func (c UDPInputConfig) Build(context operator.BuildContext) ([]operator.Operato
 		return nil, fmt.Errorf("failed to resolve listen_address: %s", err)
 	}
 
+	encoding, err := c.Encoding.Build(context)
+	if err != nil {
+		return nil, err
+	}
+
+	splitFunc, err := c.Multiline.Build(context, encoding.Encoding, true)
+	if err != nil {
+		return nil, err
+	}
+
 	udpInput := &UDPInput{
 		InputOperator: inputOperator,
 		address:       address,
-		buffer:        make([]byte, 8192),
+		buffer:        make([]byte, DefaultMaxLogSize),
```

Comment:
Should this be `buffer:        make([]byte, int(c.MaxLogSize))`?

Reply:
Or if the point is to be able to buffer multiple logs, then perhaps use a multiple of `c.MaxLogSize`? (And maybe this becomes a configurable value later?)

Reply:
This is tricky as we should read full UDP packet at once, since rest of data is going to be loss:
https://stackoverflow.com/a/24705791

I will set it to 64K as it is maximum possible udp packet size